communication/src/AsyncTlsConnection.cpp

```diff
@@ -283,8 +287,10 @@ void AsyncTlsConnection::initClientSSLContext(NodeNum destination) {
   ssl_context_.set_verify_mode(asio::ssl::verify_peer);
 
   fs::path path;
+  fs::path cert_path;
   try {
-    path = fs::path(config_.certificatesRootPath_) / fs::path(std::to_string(config_.selfId_)) / "client";
+    path = fs::path(config_.certificatesRootPath_) / fs::path(std::to_string(config_.selfId_));
+    if (bftEngine::ReplicaConfig::instance().enableMultiplexChannel == false) path = path / fs::path("client");
```

Comment:
Condition in form 
if (!bftEngine::ReplicaConfig::instance().enableMultiplexChannel)
is preferable.

Reply:
done